index.html

```diff
@@ -1177,119 +1177,305 @@ <h2 id="relationship-to-permissions-policy">
         permission=] or by some user agent policy.
       </p>
     </section>
-    <section data-cite="webdriver">
+    <section>
       <h2 id="automation">
         Automated testing
       </h2>
       <p>
         For the purposes of user-agent automation and application testing, this document defines
-        the following <a>extension commands</a> for the [[WebDriver]] specification. It is OPTIONAL
-        for a user agent to support <a>extension commands</a> commands.
+        extensions to the [[WebDriver]] and [[WebDriver-BiDi]] specifications.
+        It is OPTIONAL for a user agent to support them.
       </p>
       <pre class='idl'>
         dictionary PermissionSetParameters {
           required PermissionDescriptor descriptor;
           required PermissionState state;
         };
       </pre>
-      <section>
-        <h3 id="set-permission-command">
-          Set Permission
+      <section data-cite="webdriver-bidi webdriver">
+        <h3 id="automation-webdriver-bidi">
+          Automated testing with [[WebDriver-BiDi]]
         </h3>
-        <table>
-          <tbody>
-            <tr>
-              <th>
-                HTTP Method
-              </th>
-              <th>
-                <a data-lt="extension command URI template">URI Template</a>
-              </th>
-            </tr>
-            <tr>
-              <td>
-                POST
-              </td>
-              <td>
-                /session/{session id}/permissions
-              </td>
-            </tr>
-          </tbody>
-        </table>
         <p>
-          The <dfn class="export" data-dfn-for="extension commands">Set Permission</dfn>
-          <a>extension command</a> simulates user modification of a {{PermissionDescriptor}}'s
-          <a>permission state</a>.
+          This document defines the following [=extension modules=] for the [[WebDriver-BiDi]] specification.
         </p>
+        <section>
+          <h4 id="extension-module-browser">
+            The browser Module
+          </h4>
+          <p>
+            The [=modules/browser=] module contains commands for
```

Comment:
I am not sure we can redefine the whole module like this. From the BiDi spec:

https://w3c.github.io/webdriver-bidi/#protocol-modules

```
An implementation may define extension modules. These must have a module name that contains a single colon ":" character. The part before the colon is the prefix; this is typically the same for all extension modules specific to a given implementation and should be unique for a given implementation. Such modules extend the local end definition and remote end definition providing additional groups as choices for the defined commands and events.
```

At least we should discuss if we need to extend that defintion.

Reply:
There's no AI on this end of the spec. The AI is to change the BiDi spec to remove the colon requirement. Therefore, resolving this comment.

Reply:
Is there an issue filed to track removing the colon requirement?

Reply:
There is no "colon requirement". Note that paragraph starts "An _implementation_ may…". This specification is not an implementation, so it doesn't apply.

However the BiDi spec currently doesn't talk about external specifications at all (and presumably couldn't do so in a binding way). However if it did, it would express the preference that third party specifications add additional modules where possible rather than adding to existing commands, events or modules, just so that it's clear what the total definition of each module is.

Reply:
Should we leave the BiDi spec as is then, w.r.t. this colon statement?

Reply:
We should only (optionally) add a recommendation about how external specifications should be authored (e.g., defining complete modules). I think we can leave the text about implementation extensions as is. It looks like there is nothing blocking on the WebDriver BiDi spec side.

Reply:
I'll resolve this here and add an AI to https://github.com/w3c/webdriver-bidi/issues/587